tests/chainer_tests/initializer_tests/test_constant.py

```diff
@@ -1,16 +1,26 @@
 import unittest
 
 from chainer import backend
-from chainer.backends import cuda
 from chainer import initializers
 from chainer import testing
-from chainer.testing import attr
 import numpy
 
 
 @testing.parameterize(*testing.product({
     'dtype': [numpy.float16, numpy.float32, numpy.float64],
 }))
+@testing.backend.inject_backend_tests(
+    None,
+    [
+        # CPU
+        {},
+        # CUDA
+        {'use_cuda': True, 'cuda_device': 0},
+        # ChainerX
+        {'use_chainerx': True, 'chainerx_device': 'native:0'},
+        {'use_chainerx': True, 'chainerx_device': 'cuda:0'},
```

Comment:
Please add `{'use_ideep': 'always}`,  `{'use_cuda': True, 'cuda_device': 1}` and `{'use_chainerx': True, 'chainerx_device': 'cuda:1'}`.